using_images/s2i_images/dot_net_core.adoc

```diff
@@ -0,0 +1,85 @@
+[[using-images-using-dot-net-core]]
+= .NET Core
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[benefits-of-using-dot-net-core]]
+== Benefits of Using .NET Core
+
+{nbsp}.NET Core is a general purpose development platform featuring automatic memory
+management and modern programming languages. It allows users to build high
+quality applications efficiently. .NET Core is available on Red Hat Enterprise
+Linux (RHEL 7) and {product-title} via certified containers. .NET Core offers:
+
+* The ability to follow a microservices-based approach, where some components are
+built with .NET and others with Java, but all can run on a common, supported
+platform in Red Hat Enterprise Linux and {product-title}.
+* The capacity to more easily develop new .NET Core workloads on Windows;
+customers are able to deploy and run on either Red Hat Enterprise Linux or
+Windows Server.
+* A heterogeneous data center, where the underlying infrastructure is capable of
+running .NET applications without having to rely solely on Windows Server.
+* Access to many of the popular development frameworks such as .NET, Java, Ruby,
+and Python from within {product-title}.
+
+{nbsp}.NET Core can produce a multi-architecture binary, including compiled user
```

Comment:
One more thing to note: the fat binary (that includes both the built source and the .NET Core runtime) is not portable across architectures. It's a native binary. The version built for RHEL, for example, won't run on Windows.